docs/examples/example_oosm_algorithm.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+===================================================
+Algorithm dealing with Out-of-Sequence measurements
```

Comment:
Could you make this title more specific to distinguish from other OOSM examples? E.g. "Handling OOSM using inverse time dynamics"